common/persistence/src/main/resources/db/migration/V3__createAndUpdateTablesForFederation.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE federation_batch_info (
+    batch_tag   varchar(20) PRIMARY KEY,
+    date        date NOT NULL,
+    status      varchar(20) NOT NULL DEFAULT 'UNPROCESSED'
+);
+
+-- TODO? Migrate current values: origin_country = DE, etc.?
+ALTER TABLE diagnosis_key
+    ADD consent_to_federation BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE diagnosis_key
+    ADD origin_country VARCHAR(2);
+ALTER TABLE diagnosis_key
+    ADD visited_countries VARCHAR(2)[];
+ALTER TABLE diagnosis_key
+    ADD report_type VARCHAR(30);
+ALTER TABLE diagnosis_key
+    ADD days_since_onset_of_symptoms INTEGER;
+
+CREATE TABLE federation_upload_key (
+    key_data bytea PRIMARY KEY,
+    rolling_period integer NOT NULL,
+    rolling_start_interval_number integer NOT NULL,
+    submission_timestamp bigint NOT NULL,
+    transmission_risk_level integer NOT NULL,
+    consent_to_federation boolean NOT NULL DEFAULT FALSE,
+    origin_country varchar (2),
+    visited_countries varchar (2) [],
+    report_type varchar(30),
+    days_since_onset_of_symptoms INTEGER
```

Comment:
@ebratter - where do you plan to store the 'batch_tag' which we used for the upload?

Reply:
Batch Tag Id is part of another opened PR. We need to merge before condensing the flyway scripts